modules/channel-web/assets/default.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
```

Comment:
Please embed them in the package, we try to avoid as much as possible having external dependencies (some clients have firewalls preventing access)